tests/python/kaolin/io/test_usd.py

```diff
@@ -121,6 +121,11 @@ def test_import_hetero_fail(self, scene_paths, out_dir, hetero_mesh_path):
         with pytest.raises(usd.NonHomogeneousMeshError):
             usd.import_meshes(hetero_mesh_path, ['/Root'])
 
+    def test_import_hetero_fail2(self, scene_paths, out_dir, hetero_mesh_path):
```

Comment:
Can we have more explicit naming for the tests ? (rather the suffix `2` which is not expressing what is the difference with the first version)

Reply:
Update test names.